manifests/charts/ztunnel/templates/_helpers.tpl

```diff
@@ -1,10 +1,4 @@
-{{- define "istio-labels" }}
-    app.kubernetes.io/name: ztunnel
-    {{- if .Release.Service }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
How important are these guards?

Reply:
They matter mostly outside of helm install contexts iirc. istioctl was failing to produce valid yaml without the guard so I added them.